CHANGELOG.md

```diff
@@ -31,6 +31,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added information regarding the `--bos-version` command line argument to man
   pages for relevant subcommands.
+- Added a ``sat swap blade`` subcommand which automates the procedure for
+  swapping compute and UAN blades.
+- Added support for River blades to the ``sat swap blade`` subcommand.
+- Updated the ``sat-swap(8)`` man page to document the new ``sat swap blade``
+  subcommand.
```

Comment:
These could probably be condensed into a single item, IMO. The man page updates should go without saying, and the second one could be assumed by the first one, and honestly is a little misleading because it sounds like it might support swapping River management NCNs.

Reply:
That's a good call. I was adding entries individually on the feature branch, it makes sense to bundle them all up here.